definition/data/sheets/AuthorisationCaseEvent.json

```diff
@@ -107,6 +109,7 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "RejectOrganisationPaymentPlan", "UserRole": "caseworker-cmc-judge", "CRUD": "R"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "AgreementRejectedByDefendant", "UserRole": "caseworker-cmc-judge", "CRUD": "R"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "AgreementCounterSignedByDefendant", "UserRole": "caseworker-cmc-judge", "CRUD": "R"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "AgreementSignedByBoth", "UserRole": "caseworker-cmc-judge", "CRUD": "R"},
```

Comment:
The legal advisor authorisation is missing. Not that this makes any functional difference but may be confusing when merging down the line

Reply:
done